html/index.html

```diff
@@ -3,11 +3,80 @@
+    <link href="https://getbootstrap.com/docs/5.3/assets/css/docs.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css"></script>
 </head>
-<body>
+<body class="p-3 m-4 border-0 bd-example bg-light">
+    <ul class="nav justify-content-end">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="/html/index.html">KIT</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="feed/index.html">My feed</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="profile/index.html">My profile</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled">log out</a>
+        </li>
+        <li>
+          <form class="d-flex" role="search">
+            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-primary" type="submit">Search</button>
+          </form>
+        </li>
+      </ul>
+
+ 
+
+<!--this part needs to be nested in a card!-->
+
+    <div class="container my-5">
+      <div class="row justify-content-center align-items-center">
+        <div class="col-md-6">
+          <div class="card">
+            <div class="card-header">
+              <h1>Keep In Touch</h1>
+            </div>
+            <div class="card-body">
+              <form>
+                <div class="mb-3">
+                  <label for="exampleInputEmail1" class="form-label">Email address</label>
+                  <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+                  <div id="emailHelp" class="form-text">We'll never share your email with anyone else.</div>
+                </div>
+                <div class="mb-3">
+                    <label for="inputPassword5" class="form-label">Password</label>
+                    <input type="password" id="inputPassword5" class="form-control" aria-describedby="passwordHelpBlock">
+                    <div id="passwordHelpBlock" class="form-text">
+                      Your password must be 8-20 characters long, contain letters and numbers, and must not contain spaces, special characters, or emoji.
+                    </div>
+                </div>
+                <div class="mb-3 form-check">
+                  <input type="checkbox" class="form-check-input" id="exampleCheck1">
+                  <label class="form-check-label" for="exampleCheck1">Remember me</label>
+                </div>
+                <button type="submit" class="btn btn-default"><a href="feed/index.html">Log in</a></button>
```

Comment:
You don't need an anchor tag in a button. The line should be 

```
<button type="submit" class="btn btn-default">Log in</button>```

Reply:
I am aware of this, this was made since the CA had no js- to be able to move to the next page. I can remove it - and next time I wil make a comment on the code.